lib/logstash/codecs/cef.rb

```diff
@@ -114,6 +117,9 @@ def decode(data)
     end
 
     yield event
+  rescue => e
+    @logger.error("Failed to decode event payload. Generating failure event with payload in message field.", :error => e.message, :backtrace => e.backtrace, :data => data)
```

Comment:
I suggest to add some kind of reference to CEF to the error log message. While checking the logs it would be easier to understand the error.
Maybe something like: "Failed to decode CEF payload." ...